lib/x86simdsort-spr.cpp

```diff
@@ -1,15 +1,15 @@
 // SPR specific routines:
-#include "avx512fp16-16bit-qsort.hpp"
+#include "x86simdsort-static-incl.h"
 #include "x86simdsort-internal.h"
 
 namespace xss {
 namespace avx512 {
     template <>
     void qsort(_Float16 *arr, size_t size, bool hasnan, bool descending)
     {
-        if (descending) { avx512_qsort<true>(arr, size, hasnan); }
+        if (descending) { x86simdsortStatic::qsort(arr, size, hasnan, true); }
```

Comment:
We needed this if statement before, but with the changes made couldn't we eliminate it and pass descending directly? The same applies to qselect and partial_qsort

Reply:
Good point. Got rid of it.